chromium_edits/129.0.6658.0/chrome/browser/shell_integration_mac.mm.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/shell_integration_mac.mm b/chrome/browser/shell_integration_mac.mm
+index c6bb768979453..c0d0abb9c78f3 100644
+--- a/chrome/browser/shell_integration_mac.mm
++++ b/chrome/browser/shell_integration_mac.mm
+@@ -79,6 +79,8 @@ bool SetAsDefaultBrowser() {
+     if (LSSetDefaultHandlerForURLScheme(CFSTR("https"), identifier) != noErr) {
+       return false;
+     }
++    LSSetDefaultHandlerForURLScheme(CFSTR("ipfs"), identifier);
```

Comment:
In this PR, one _does_ actually need to pay attention to the chromium_edits/ folder. 

As you can tell from the `_mac.mm` suffix, this file is specific to Mac OS.